doc/README.md

```diff
@@ -42,59 +73,163 @@ services:
-The command will start a container, which will monitor SASS directories for changes and will compile them to CSS files when changes are detected.
-Note that if you have mounted the code to `/src/`, you need to restart the servers for these updates to take effect.
+Some parts are still easier to do without containers, tests and translations are such.
+Therefore they are run and updated outside the containers, within a virtual environment.
+
+### Prerequisities
 
-After completing the changes to these types of assets, rerun the `drone exec` to build production versions of the CSS and JavaScript files, which should be added to the git for now (i.e. as long as there is no real CI/CD pipeline for A+).
+- Firefox browser installed
+- python3 installed
+- [Python module venv (or virtualenv)](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv) installed
 
-## Running tests
+1. Create a virtual environment for a-plus and activate it (e.g. in the project root):
 
-Before creating a pull-request, you should verify the code is still working against existing tests.
-Even better, you should write more tests, when possible!
-Nevertheless, you can do that on the host with the command `venv/bin/python manage.py test` (this assumes you did the the virtual environment installation from above).
+```
+python3 -m venv venv_aplus
+source venv_aplus/bin/activate
+```
 
-## Selenium integration tests
+2. Install necessary requirements for development:
 
-Selenium tests are designed to run against a certain database state and service ports.
+```
+pip3 install --upgrade pip setuptools wheel
+pip3 install -r requirements.txt
+```
+
+3. Create a `local_settings.py` -file at `a-plus/local_settings.py`.
+There is an example file `local_settings.example.py`, which you can use as a starting point.
+
+4. Ensure your `local_settings.py` contains at least the following lines:
+
+```
+DEBUG = True
+BASE_URL = 'http://localhost:8000/'
+```
+
+5. Run the existing django migrations:
+
+```
+python manage.py migrate
+```
+
+6. You can now run the local version without a container:
+
+```
+python manage.py runserver
+```
+
+If you need to be able to login or use the admin-page while running the code without a container, you can create a superuser by following [these instructions](https://docs.djangoproject.com/en/3.0/intro/tutorial02/#introducing-the-django-admin). 
+
+### Unit tests
+
+1. Run the unit tests:
+
+```
+python manage.py test
+```
+
+### Selenium integration tests
+
+Selenium tests check end-to-end -functioning from a user-perspective.
 The tests will automatically open and direct Firefox browser windows.
-Currently, the tests depend on Unix type shell.
-All the following commands assume that the project virtualenv has been activated:
+Currently, the tests depend on a Unix type shell.
+
+1. Install the necessary requirements:
+
+```
+pip install -r requirements_testing.txt
```

Comment:
I think we could try to be consistent and use always `pip3` instead of `pip`. In virtual environment, `pip` is alias for `pip3` (assuming python3 installation), but that is not the cases out of the environment. To ensure there is no mistakes, it might be better to always use `pip3` in the guide.

Reply:
Good point, that has now been changed